deploy/olm/ubi-images/Dockerfile.acmesolver

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The Jetstack cert-manager contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM quay.io/jetstack/cert-manager-acmesolver:v0.15.0-alpha.0 as source
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+COPY --from=source /app /app
+# Required Licenses
+ADD https://raw.githubusercontent.com/jetstack/cert-manager/master/LICENSE /licenses/LICENSE
+ADD https://raw.githubusercontent.com/jetstack/cert-manager/master/LICENSES /licenses/LICENSES
```

Comment:
Again we should fix this up - can we include the LICENSE files in our regular docker images that we publish?

Reply:
Sounds sane!

Reply:
Sounds like a good idea, probably to add in a later iteration, if we change that here the builds we now run fail certification, kind of coming back to a circular dependancy problem again here...